app.py

```diff
@@ -55,14 +55,30 @@ def automate_vehicle():
     return "a"
 
 
-@app.route("/tree")
-def tree():
+@app.route("/init_tree")
```

Comment:
Why setting a route to initialise the tree? Can't it be done when the server is starting?

Reply:
two reasons why : 
1. Creating an Object every time we start up a server even when its not required. (u only geocode trees once)
2. Anyways i'm moving the Initiation step to add_tree itself. I had put in separate bcos I was facing some errors.